pyzscaler/zia/locations.py

```diff
@@ -1,21 +1,29 @@
-from restfly.endpoint import APIEndpoint
 from box import BoxList
-from pyzscaler.utils import snake_to_camel
+from restfly.endpoint import APIEndpoint
+
+from pyzscaler.utils import Iterator, snake_to_camel
 
 
 class LocationsAPI(APIEndpoint):
-    def list_locations(self):
+    def list_locations(self, **kwargs):
         """
-        Returns a list of configured locations.
+        Return list of locations.
 
+        Keyword Args:
+            **max_items (int, optional):
+                The maximum number of items to return before stopping iteration.
+            **max_pages (int, optional):
+                The maximum number of pages to request before throwing stopping iteration.
```

Comment:
Quality/Grammar: The maximum number of pages to request before stopping iteration.